### ISSUE TITLE
`localns` arguments are not longer accepted.

### ISSUE THREAD

**Issue by commonism** [pydantic/pydantic#5383]
### Initial Checks

- [X] I confirm that I'm using Pydantic V2 installed directly from the `main` branch, or equivalent

### Description

I used localns to create nested models dynamically using pydanticv1.
Define all models, `….update_forward_refs({"ModelNameA":ModelA","ModelNameA":ModelA",})` for all of them in the proper order and it works.

How to without localns using pydanticv2?

### Python, Pydantic & OS Version

```Text
pydantic version: 2.0a1
                 install path: ~/venv/openapi3/lib/python3.10/site-packages/pydantic
               python version: 3.10.6 (main, Mar 10 2023, 10:55:28) [GCC 11.3.0]
                     platform: Linux-5.19.0-35-generic-x86_64-with-glibc2.35
     optional deps. installed: ['email-validator', 'typing-extensions']
```

**Comment by dmontagu** [comment]
Yeah, I realized we had eliminated this and was concerned someone might want the ability to specify the localns directly. I think we can re-add this into `model_rebuild`, I'll take a stab at it.

**Comment by dmontagu** [comment]
@commonism the "fix" for this from #5385 is included now in the just-released `pydantic==2.0a2`. Please let us know if you have any further issues with this.
